src/igvfd/types/sample.py

```diff
@@ -57,6 +57,25 @@ def concat_numeric_and_units(numeric, numeric_units, no_numeric_on_one=False):
         return f'{numeric} {numeric_units}s'
 
 
+def compose_summary_sample_term_phrase(sample_term_term_names: list, cap_number: int) -> str:
```

Comment:
I see that below, as input arguments, you pass in `sorted(sample_term_term_names)` both for the term name and the targeted term names. Maybe it would be slightly cleaner if this function starts by doing `sample_term_term_names = sorted(sample_term_term_names)` and then the input argument can be passed in any order, the function will handle the ordering.